Public Relations Approval Process.md

```diff
@@ -0,0 +1,8 @@
+**Publication Approval Process**
+
+1.  Team member drafts content for release (this includes posters, presentations, manuscripts, and anything that will be released publicly that is branded KBase).
```

Comment:
Could you define what "branded KBase" means, exactly? Publications are not branded anything.